### ISSUE TITLE
Add atributte "DistRead"

### ISSUE THREAD

**Issue by NickeLogan** [ottools/ItemEditor#7]
You could add the attribute "Distread", it is very useful to create the cards that are seen with the "look" option. I do not know why this option was removed after the 0.3.9 version of the itemeditor.

**Comment by ghost** [comment]
yeah, I miss these option too, the workaround now is to edit items.xml with `<attribute key="allowDistRead" value="true" />`
